server/config/config.js

```diff
@@ -0,0 +1,47 @@
+const dotenv = require("dotenv");
+dotenv.config();
+
+module.exports = {
+    "development": {
```

Comment:
Do we need the environment here at all?

Reply:
Probably not. We should assume that the env file will define the environment; i.e. if we are running in the dev environment the env file will contain dev values, if we are running in the prod environment, the env file will contain prod values etc.

Reply:
That's what I'm thinking.